src/test/scala/org/broadinstitute/dsde/workbench/sam/google/GoogleExtensionRoutesSpec.scala

```diff
@@ -307,7 +307,9 @@ class GoogleExtensionRoutesSpec extends GoogleExtensionRoutesSpecHelper with Sca
     // A duplicate call should return OK and an empty response
```

Comment:
This comment looks to be out of date given these changes.

Reply:
Thanks! Changed to reflect ✨reality✨